apps/explorer/src/components/common/Button/index.tsx

```diff
@@ -254,4 +138,17 @@ export const ButtonBase = styled(ThemeWrappedButtonBase)<ButtonBaseProps>`
   &[disabled] {
     pointer-events: none;
   }
+
+  ${({ variant }) => getButtonVariantStyles(variant)}
+  ${({ size }) => getButtonSizeStyles(size)}
+
+  > svg {
+    margin: 0 0.5rem 0 0;
+  }
 `
+
+export const Button: React.FC<ButtonBaseProps> = ({ children, ...restProps }) => (
+  <ButtonBase {...restProps}>{children}</ButtonBase>
+)
+
+export default Button
```

Comment:
`Button` export is not use actually, should be deleted.
Also, we don't use default exports. Yes, there are lots of legacy code with them, and we try to refactor it.

Reply:
Addressed this now.